Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Beam/Merchant/MerchantServiceUsageConfig.hs

```diff
@@ -52,7 +52,8 @@ data MerchantServiceUsageConfigT f = MerchantServiceUsageConfigT
     issueTicketService :: B.C f IssueTicketService,
     getExophone :: B.C f CallService,
     updatedAt :: B.C f UTCTime,
-    createdAt :: B.C f UTCTime
+    createdAt :: B.C f UTCTime,
+    verificationProvidersPriorityList :: B.C f (Maybe [VerificationService])
```

Comment:
Lets make it NonMaybe as it is config, we can add default as ["Idfy"]

Reply:
modified